amd_openvx_extensions/amd_rpp/source/tensor/Spectrogram.cpp

```diff
@@ -29,7 +29,7 @@ struct SpectrogramLocalData {
     RppPtr_t pDst;
     bool centerWindows;
     bool reflectPadding;
-    RpptSpectrogramLayout spectrogramLayout;
+    vxTensorLayout spectrogramLayout;
```

Comment:
Is this added new to the framework?

Reply:
vxTensorLayout is extended to support new rppt layouts which includes the spec layout so we are using vxTensorLayout to specify the layout